code/modules/clothing/suits/labcoat.dm

```diff
@@ -130,6 +130,14 @@
 	icon_closed = "labcoat_surgeon"
 	armor = list(melee = 0, bullet = 0, laser = 0,energy = 0, bomb = 0, bio = 60, rad = 0)
 
+/obj/item/clothing/suit/storage/toggle/labcoat/em_phys
```

Comment:
Unwieldy typepath. This isn't a blocking issue but it'd be nicer if this was made more sane.